great_expectations/datasource/fluent/spark_datasource.py

```diff
@@ -57,6 +58,12 @@ class _SparkDatasource(Datasource):
     spark_config: Union[SparkConfig, None] = None
     force_reuse_spark_context: bool = True
 
+    @staticmethod
+    def _update_asset_forward_refs(asset_type: Type[_DataAssetT]) -> None:
+        # Only update forward refs if pyspark types are available.
+        if pyspark:
```

Comment:
@anthonyburdi I am not sure whether or not this will work -- it might need to be `pyspark.<something>`.  Thanks.

Reply:
Good catch! I missed the second pyspark. Fixing.

Reply:
Fixed!